src/components/interaction-designer/blocks/BlockToolbar.vue

```diff
@@ -8,13 +8,13 @@
           v-b-tooltip.hover="trans('flow-builder.deselect-block')"
           :icon="['far', 'check-circle']"
           class="cursor-pointer text-info"
-          @click="isEditable && block_deselect({ blockId: block.uuid })" />
+          @click.stop="isEditable && block_deselect({ blockId: block.uuid })" />
```

Comment:
took the opportunity for fix those event propagations

Reply:
where they causing a block to be selected/unselected?

Reply:
@bulatgab , yes!